### PR TITLE
Added Danish language support

### PR DESCRIPTION
**Pull request by Andersbiha**
Danish language support.

Since AM, PM and ordinal number notation isn't used in Danish i kept the defaults.

### CODE REVIEW

File: src/locale/dk/build_format_locale/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+function buildFormatLocale () {
+  var months3char = ['jan', 'feb', 'mar', 'apr', 'maj', 'jun', 'jul', 'aug', 'sep', 'okt', 'nov', 'dec']
+  var monthsFull = ['januar', 'februar', 'marts', 'april', 'maj', 'juni', 'juli', 'august', 'september', 'oktober', 'novermber', 'december']
+  var weekdays2char = ['sø', 'ma', 'ti', 'on', 'to', 'fr', 'lø']
+  var weekdays3char = ['søn', 'man', 'tir', 'ons', 'tor', 'fre', 'lør']
+  var weekdaysFull = ['søndag', 'mandag', 'tisdag', 'onsdag', 'torsdag', 'fredag', 'lørdag']
+  var meridiemUppercase = ['AM', 'PM']
+  var meridiemLowercase = ['am', 'pm']
+  var meridiemFull = ['a.m.', 'e.m.']
```
Comment (LeshaKoss) [inline src/locale/dk/build_format_locale/index.js]: > 'e.m.'

@Andersbiha is it a typo?

**Comment by LeshaKoss** [comment]
@Andersbiha thank you for your contribution! Please fix failing tests https://travis-ci.org/date-fns/date-fns/builds/184494140#L5775

**Comment by Andersbiha** [comment]
Argh - Just realized i'd been pushing from the wrong branch.
I just repushed with the changes needed for the tests to succeed, thanks for the patience ;)

**Comment by DeMuu** [comment]
I think the locale short name should be changed to DA

the [ISO 639-2](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) list states the 2 letter code as DA :)

**Comment by LeshaKoss** [comment]
@Andersbiha thanks again for contribution! 🇩🇰 Danish locale was released with [1.21.0](https://github.com/date-fns/date-fns/releases/tag/v1.21.0) (diff: https://github.com/date-fns/date-fns/compare/v1.20.1...v1.21.0)

**Pull request merged**